LocaleUtils/index.js

```diff
@@ -7,7 +7,10 @@ export async function assertNoEnglishLeaks(mockFile, viewText, noTranslationCont
   }
   // Exclude all elements with a class of no-translate. Example phone numbers
   if (noTranslationContent.length) {
-    viewText = viewText.split(noTranslationContent).join('');
+    //replace each noTranslationContent
+    for (var i = 0; i < noTranslationContent.length; i++) {
```

Comment:
Can we use `forEach` here since `noTranslationContent` is going to be an Array? IMO this is a bit easier to follow.